tests/foreman/ui/test_repository.py

```diff
@@ -549,6 +549,106 @@ def test_positive_sync_custom_repo_docker(self):
                     self.assertTrue(self.prd_sync_is_ok(repo_name))
 
     @run_only_on('sat')
+    @tier2
+    def test_positive_resynchronize_rpm_repo(self):
+        """Check that that repository content is resynced after packages were
+        removed from repository
+
+        @id: dc415563-c9b8-4e3c-9d2a-f4ac251c7d35
+
+        @Assert: Repository have updated non-zero package counts
+
+        @CaseLevel: Integration
+
+        @BZ: 1318004
+        """
+        with Session(self.browser) as session:
+            repo = entities.Repository(
+                url=FAKE_1_YUM_REPO,
+                content_type='yum',
+                product=self.session_prod,
+            ).create()
+            self.setup_navigate_syncnow(
+                session,
+                self.session_prod.name,
+                repo.name,
+            )
+            self.assertTrue(self.prd_sync_is_ok(repo.name))
+            # Check packages number
```

Comment:
Move
```
self.repository.search_and_click(repo.name)
number = self.repository.find_element(locators['repo.fetch_packages'])
return int(number.text)
```
to separate helper. We should avoid working with locators in tests as more as we can